mlrun/db/httpdb.py

```diff
@@ -259,6 +259,12 @@ def connect(self, secrets=None):
                     f"warning!, server ({server_cfg['namespace']}) and client ({config.namespace})"
                     " namespace don't match"
                 )
+            config.ce.mode = server_cfg.get("ce_mode") or config.ce.mode
+            if "ce_mode" in server_cfg and server_cfg["ce_mode"] != config.ce.mode:
+                logger.warning(
+                    f"warning!, server ({server_cfg['ce_mode']}) and client ({config.ce.mode})"
+                    " CE mode don't match"
+                )
```

Comment:
This will always be true, as you are setting `config.ce.mode` to server_cfg before.
```suggestion
            if config.ce.mode and config.ce.mode != server_cfg.get("ce_mode",""):
                logger.warning(
                    f"warning!, server ({server_cfg['ce_mode']}) and client ({config.ce.mode})"
                    " CE mode don't match"
                )
            config.ce.mode = server_cfg.get("ce_mode") or config.ce.mode
```